docs/topics/mta-rn-upgrade-cli.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-cli_{context}"]
+= Upgrading the CLI
+
+When you update the CLI of {ProductName} ({ProductShortName}) from version 6.0.0 to version {ProductVersion}, be sure to set the correct value of `rwx_supported`.
+
+.Procedure
+
+. Follow the installation instructions in the link:{ProductDocUserGuideURL}/index#installing-web-console-or-cli-tool_cli-guide [CLI guide].
+. {ProductShortName} {ProductVersion} includes a new parameter, `rwx_supported`.
+
+** If you set it to `true`, only users with RWX permissions can run {ProductShortName}.
+** If you set it to `false`, other users can run {ProductShortName}.
```

Comment:
This is not an obligation to configure this parameter. If user will not update tackle CR with it - it will just work same way as if it was set to true. It is obligatory to configure it only if user wants to disable it.